goutubedl.go

```diff
@@ -217,13 +217,14 @@ var TypeFromString = map[string]Type{
 
 // Options for New()
 type Options struct {
-	Type               Type
-	PlaylistStart      uint   // --playlist-start
-	PlaylistEnd        uint   // --playlist-end
-	Downloader         string // --downloader
-	DownloadThumbnail  bool
-	DownloadSubtitles  bool
-	DownloadSections   string // --download-sections
+	Type              Type
+	PlaylistStart     uint   // --playlist-start
+	PlaylistEnd       uint   // --playlist-end
+	Downloader        string // --downloader
+	DownloadThumbnail bool
+	DownloadSubtitles bool
+	DownloadSections  string // --download-sections
+
```

Comment:
Maybe remove this extra new line? then i think the rest for the style change will go back to as before